<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting2Widget</class>
 <widget class="QWidget" name="Setting2Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="324,312,110">
     <item row="0" column="1">
      <widget class="CustomComboBox" name="reticleShapeComboBox"/>
     </item>
     <item row="0" column="0">
      <widget class="CustomLabel" name="reticleShapeLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="CustomComboBox" name="weatherModeComboBox"/>
     </item>
     <item row="4" column="1">
      <widget class="CustomComboBox" name="antiJammingModeComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="CustomLabel" name="weatherModeLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="CustomLabel" name="buzzerLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="CustomComboBox" name="buzzerComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="CustomLabel" name="daynNightLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="CustomLabel" name="antiJammingModeLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="CustomComboBox" name="daynNightComboBox"/>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>CustomLabel.h</header>
  </customwidget>
  <customwidget>
   <class>CustomComboBox</class>
   <extends>QComboBox</extends>
   <header>CustomComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
